packages/backend/src/database/migrations/20230714124708_user_attributes.ts

```diff
@@ -0,0 +1,61 @@
+import { Knex } from 'knex';
+
+const userAttributesTableName = 'user_attributes';
+const organizationMemberUserAttributes = 'organization_member_user_attributes';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(userAttributesTableName))) {
+        await knex.schema.createTable(userAttributesTableName, (table) => {
+            table
+                .uuid('user_attribute_uuid')
+                .primary()
+                .defaultTo(knex.raw('uuid_generate_v4()'));
+            table.string('name').notNullable();
+            table.string('description').nullable();
+
+            table
+                .timestamp('created_at', { useTz: false })
+                .notNullable()
+                .defaultTo(knex.fn.now());
+            table
```

Comment:
If we use `organization_id` here we can enforce that members are only assigned to attributes in their orgs. See below